modules/rosa-create-cluster-admins.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/creating-first-rosa-cluster.adoc
+
+
+[id="rosa-create-dedicated-cluster-admins.adoc"]
+= Granting cluster-admin access
+
+As the user that created the cluster, add the `cluster-admin` user role to your account to have the maximum administrator privileges. These privileges are not automatically assigned to your user account when you create the cluster.
+
+Additionally, only the user who created the cluster can grant cluster access to other `cluster-admin` or `dedicated-admin` users. `dedicated-admin` users have fewer privileges. As a best practice, limit the number of `cluster-admin` users.
+
+.Prerequisites
+
+* You have added an Identity Provider (IDP) to your cluster.
```

Comment:
I don't think "Identity Provider" needs to be capitalized here as a generic term.

Reply:
Done.